### ISSUE TITLE
[vcpkg docs] Improve vcpkg command related documents

### ISSUE THREAD

**Issue by Huang88888888Fei** [microsoft/vcpkg#12811]
****

**Is your feature request related to a problem? Please describe.**
The download and compilation time of vcpkg is relatively long. So during this waiting period, I would rather go play a game, have a cup of coffee or do some other work. But, but, but! ! ! 
When the network is not good, vcpkg will always interrupt because it has not downloaded the complete package, which is very annoying! ! ! This makes me have to stare at the screen all the time, watching vcpkg from downloading to compiling and installing, in case any package download fails and the compilation is interrupted.

**Proposed solution**
So my suggestion is: at the beginning of the installation, list all dependent packages and their download links, and indicate which packages already exist. 
So I can find a place where the network is stable, download all the dependent packages, put them in the specified directory, and then start compiling directly. In this way, I can go to drink my coffee without worries.

**Describe alternatives you've considered**
I think the file [vcpkg/scripts/cmake/vcpkg_download_distfile.cmake](https://github.com/microsoft/vcpkg/blob/21b56d99293aa364964b99946847986c5276c3b8/scripts/cmake/vcpkg_download_distfile.cmake) should be modified, but I don't know how to do and hope you can give me some guidance. On the other hand, I would appreciate it if you can take my advice and update the source code.

**Comment by StarGate-One** [comment]
Have you tried **`vcpkg install port port port --only-downloads`** to download all the requested port packages and their dependencies then follow-up with **`vcpkg install port port port`** to actually install them?

**Comment by JackBoosY** [comment]
You may need to use the argument `--only-downloads`.

**Comment by Huang88888888Fei** [comment]
@StarGate-One @JackBoosY Thank You! It does work! I think I should read a product manual before I use vcpkg. And if I want to learn more parameter usage, which document should I check? Thank you very much!

**Comment by JackBoosY** [comment]
@Huang88888888Fei Sadly, our documents are not complete and we will improve them in the future.

**Comment by Huang88888888Fei** [comment]
> @Huang88888888Fei Sadly, our documents are not complete and we will improve them in the future.

Yeah, looking forward to it! Thank you again. I think I can close this issue.

**Comment by JackBoosY** [comment]
Please keep this issue open util we solve it.

**Comment by GopherJ** [comment]
Also it seems that vcpkg uses `raw.githubusercontent.com` while building boost, due to GFW's limitation, it always fail. I'm wondering if there is a solution for this?

**Comment by JackBoosY** [comment]
@GopherJ Waiting for the mirror PR merge, I will create a Chinese mirror.
